apiv2/db_import/tests/test_db_alignment_import.py

```diff
@@ -0,0 +1,132 @@
+        for alignment in run.alignments:
+            expected = next(expected_iter)
+            if "run_id" not in expected:
+                expected["run_id"] = run.id
+            verify_model(alignment, expected)
+
+
+def test_import_alignments_stale_deletion(
+    sync_db_session: Session,
+    verify_dataset_import: Callable[[list[str]], models.Dataset],
+    verify_model: Callable[[Base, dict[str, Any]], None],
+    expected_alignments: list[dict[str, Any]],
+) -> None:
+    populate_alignments(sync_db_session)
+    populate_stale_run(sync_db_session)
+    populate_stale_alignments(sync_db_session)
+    sync_db_session.commit()
+
+    actual = verify_dataset_import(import_alignments=True)
+    expected_iter = iter(expected_alignments)
+    for run in sorted(actual.runs, key=lambda x: x.name):
+        for alignment in run.alignments:
+            expected = next(expected_iter)
+            if "run_id" not in expected:
+                expected["run_id"] = run.id
+            verify_model(alignment, expected)
+
+
+def test_import_per_section_alignment_parameters(
+    sync_db_session: Session,
+    verify_dataset_import: Callable[[list[str]], models.Dataset],
+    verify_model: Callable[[Base, dict[str, Any]], None],
+    expected_per_section_alignment_parameters: list[dict[str, Any]],
+) -> None:
+    populate_per_section_alignment_parameters(sync_db_session)
+    sync_db_session.commit()
+    actual = verify_dataset_import(import_per_section_alignment_parameters=True)
+    expected_iter = iter(expected_per_section_alignment_parameters)
+    for run in sorted(actual.runs, key=lambda x: x.name):
+        for alignment in run.alignments:
+            for per_section_alignment_parameters in alignment.per_section_alignment_parameters:
+                expected = next(expected_iter)
+                if "alignment_id" not in expected:
+                    expected["alignment_id"] = alignment.id
+                verify_model(per_section_alignment_parameters, expected)
+
+
+def test_import_per_section_alignment_parameters_stale_deletion(
+    sync_db_session: Session,
+    verify_dataset_import: Callable[[list[str]], models.Dataset],
+    verify_model: Callable[[Base, dict[str, Any]], None],
+    expected_per_section_alignment_parameters: list[dict[str, Any]],
+) -> None:
+    populate_per_section_alignment_parameters(sync_db_session)
+    sync_db_session.commit()
+    actual = verify_dataset_import(import_per_section_alignment_parameters=True)
+    expected_iter = iter(expected_per_section_alignment_parameters)
+    for run in sorted(actual.runs, key=lambda x: x.name):
+        for alignment in run.alignments:
+            for per_section_alignment_parameters in alignment.per_section_alignment_parameters:
+                expected = next(expected_iter)
+                if "alignment_id" not in expected:
+                    expected["alignment_id"] = alignment.id
+                verify_model(per_section_alignment_parameters, expected)
```

Comment:
so if I'm reading this test (`expected_per_section_alignment_parameters` only returns a single object) and our test data (there are two items in the `per_section_alignment_parameters` list in the json file) properly, I think we should expect to have *two* per section alignment parameters in our db, but this test is only getting a single row? What's going on here?

Reply:
Updated the populate db script accordingly